Extensions/fonts/freetype.lisp

```diff
@@ -0,0 +1,441 @@
+(defpackage :clim-freetype
+  (:use :cl)
+  (:export #:freetype-font-renderer))
+
+(in-package :clim-freetype)
+
+(declaim (optimize (speed 0) (safety 3) (debug 3)))
+
+(setq clim:*default-server-path* '(:clx :font-renderer clim-freetype:freetype-font-renderer))
+
+(defparameter *freetype-font-scale* 26.6)
+
+(defvar *enable-autohint* t)
+
+;; The freetype2 library doesn't expose this function, so we add it here.
+(cffi:defcfun ("FT_Property_Set" ft-property-set) freetype2-types:ft-error
+  (library freetype2-types:ft-library)
+  (mod-name :string)
+  (prop-name :string)
+  (value :pointer))
+
+(defclass freetype-font-renderer (clim-clx::font-renderer)
+  ())
+
+(defmethod initialize-instance :after ((obj freetype-font-renderer) &key)
+  #+nil (cffi:with-foreign-objects ((v :int))
```

Comment:
why is it disabled? please add comment. also `#+(or)` is the correct way to do that.

Reply:
It's disabled because it doesn't seem to work the way the documentation suggests it does. I'll probably remove the entire function before the merge.